goimports/runner.bash.template

```diff
@@ -11,8 +11,10 @@ mkdir -p "src/$PREFIX_DIR_PATH"
 ln -snf "$BUILD_WORKSPACE_DIRECTORY" "src/$PREFIX_DIR_PATH/$PREFIX_BASE_NAME"
 
 GOPATH=$(pwd -P)
+GOCACHE="$(mktemp -d)"
```

Comment:
Don't we want to allow `GOCACHE` to preserve its value in case its already set?

Reply:
I think we don't to avoid depending on external state as much as we can. I assume `goimports` shouldn't be downloading anything if the directory is empty?